### DISCUSSION TITLE
Different Priors of ROI for different Media Channel

### DISCUSSION THREAD

**Discussion by shubhambooking2** [google/meridian#133] [General]
How individual priors for ROI can be set for different media channels. I tried a couple of way specifying indvidual ROI priors for media channels, however because of the assertion present at the below line, it does not allow to specify individual channel ROI priors. 

https://github.com/google/meridian/blob/e79db319a285a90cab0dcc0dcf40acf023936736/meridian/model/model.py#L196

Is there a specific reason behind not allowing for this scenario?

**Comment by alexwakim** [comment]
How to set the ROI prior for individual media channels is described here: https://developers.google.com/meridian/docs/user-guide/configure-model#set-priors

If you still get the error after checking out the link, could you reply with the code you're using to set the prior?

**Reply by shubhambooking2** [reply]
Thanks alex, this helped
